bootcamp/tutorials/integration/haystack/full_text_search_with_milvus_and_haystack.ipynb

```diff
@@ -0,0 +1,602 @@
+{
```

Comment:
<p>making it an essential complementary capability -&gt;</p><p><strong>making it a useful complement to semantic search.</strong></p><p> </p><p>&nbsp;<a href="https://docs.zilliz.com/docs/register-with-zilliz-cloud" target="_blank">trying Zilliz Cloud</a> (fully managed Milvus)&nbsp;for free.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>